seatunnel-core/seatunnel-core-flink-sql/pom.xml

```diff
@@ -60,6 +60,16 @@
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.ververica</groupId>
+            <artifactId>flink-sql-connector-mysql-cdc</artifactId>
+            <version>${flink.cdc.version}</version>
+        </dependency>
+        <dependency>
```

Comment:
I am not sure if it is ok to involve flink-runtime-web dependency. Maybe it's good for debug. cc @ruanwenjun